app.lua

```diff
@@ -126,8 +134,18 @@ function app:handle_404()
 end
 
 function app:handle_error(err, trace)
+    -- self.current_user is not available here.
+    local current_user = package.loaded.Users:find(self.session.username)
```

Comment:
I really don't understand why this is necessary. 🤷‍♂️

Reply:
Yeah, I think it has something to do with certain objects from a before_filter not being available? I tried on master and couldn't log `self.current_user.username`